### PR TITLE
Removed more Eris Mentions

### PR DESCRIPTION
**Pull request by SirSiggles**
## About The Pull Request

Removed more Eris mentions. (Ironhammer and Moebius, a few NeoTheology mentions) and added a few lines to the propaganda chip.
## Why It's Good For The Game
We're not Eris.

## Changelog
:cl:
add: New Propaganda Chip Lines.
tweak: Fixed mentions of Ironhammer, Moebius, NeoTheology and a mention of the corporate Wars.
/:cl:

### CODE REVIEW

File: code/game/objects/items/devices/holowarrant.dm [hunk 1]
```diff
@@ -9,8 +9,8 @@
 	throw_range = 10
 	slot_flags = SLOT_BELT
 	req_access = list(list(access_heads, access_security))
-	var/boss_name = "Ironhammer Security"
-	var/station_name = "CEV Eris"
+	var/boss_name = "Aegis Security"
+	var/station_name = "CEV Northern Light"
```
Comment (Greenjoe12345) [inline code/game/objects/items/devices/holowarrant.dm:13]: shouldn't it be NEV Northern Light?

File: code/game/objects/items/weapons/storage/belt.dm [hunk 2]
```diff
@@ -149,11 +149,11 @@
 	)
 
 /obj/item/weapon/storage/belt/tactical/ironhammer
-	name = "ironhammer tactical belt"
+	name = "aegis tactical belt"
 	icon_state = "tactical_ironhammer"
 
 /obj/item/weapon/storage/belt/tactical/neotheology
-	name = "neotheologian tactical belt"
+	name = "mekhanite tactical belt"
```
Comment (EvilJackCarver) [inline code/game/objects/items/weapons/storage/belt.dm:156]: inconsistent adjective/demonym, down below uses "mekhane" as an adjective/demonym whereas this is the only use of "mekhanite"

File: code/modules/projectiles/guns/mods/mods.dm [hunk 3]
```diff
@@ -142,7 +142,7 @@
 //Adds radiation damage to .35 rounds. Acquired through telecrystal uplink
 /obj/item/weapon/gun_upgrade/mechanism/glass_widow
 	name = "Syndicate \"Glass Widow\" infuser"
-	desc = "An old technology from the Corporate Wars, used to make formerly useless civilian-grade weaponry into something much more lethal."
+	desc = "An illega modification, used to make formerly useless civilian-grade weaponry into something much more lethal."
```
Comment (EvilJackCarver) [inline code/modules/projectiles/guns/mods/mods.dm:145]: typo

File: code/game/objects/items/devices/holowarrant.dm (hunk 1 above)
Comment (EvilJackCarver) [inline code/game/objects/items/devices/holowarrant.dm:13]: NEV Northern Light

**Comment by SirSiggles** [comment]
I'll handle 'em now!

**Pull request merged**